src/components/connections/BucketNode.ts

```diff
@@ -35,18 +36,18 @@ export class BucketNode implements INode {
 
   // get all the measurements
   public async getChildren(): Promise<INode[]> {
-    let queryEngine: QueryEngine = new QueryEngine(this.outputChannel);
-    let query = `import "influxdata/influxdb/v1"
-    v1.measurements(bucket:"${this.bucket}")`;
-    if (this.conn.version === InfluxConnectionVersion.V1) {
-      query = "show measurements";
+    try {
+      const msg = `Getting measurements for bucket: ${this.bucket}`;
+      this.outputChannel.show();
+      this.outputChannel.appendLine(`${now()} - ${msg}`);
+
+      const results = await Queries.measurements(this.conn, this.bucket);
```

Comment:
one thing I was unable to find is the timeout for the request. Are we applying any to http requests?

Reply:
axios have default timeouts